substrate/utils/frame/benchmarking-cli/src/storage/cmd.rs

```diff
@@ -96,10 +103,12 @@ pub struct StorageParams {
 	#[arg(long, default_value_t = 1)]
 	pub warmups: u32,
 
-	/// The `StateVersion` to use. Substrate `--dev` should use `V1` and Polkadot `V0`.
+	/// The `StateVersion` to use.
+	///
+	/// Substrate `--dev` should use `v1` and Polkadot `v0`.
 	/// Selecting the wrong version can corrupt the DB.
-	#[arg(long, value_parser = clap::value_parser!(u8).range(0..=1))]
-	pub state_version: u8,
```

Comment:
Hm, i think fixing the docs makes more sense here since its a breaking change otherwise.

Reply:
Yes please, or implement a custom parser that accepts all of "0", "v0", "V0"

Reply:
My initial thought was, that since this is for benchmark only, which means it's not a critical core feature and can be fixed easily, I chose less code instead of maintaining the compatibility. But happy to amend it if you persist.

Reply:
Changed to just fix the docs in the latest commit.